test/functional/ui/syntax_conceal_spec.lua

```diff
@@ -3,6 +3,14 @@ local Screen = require('test.functional.ui.screen')
 local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
 local insert = helpers.insert
 
+do
+  clear()
+  if helpers.os_name() == 'windows' then
+    pending('FIXME: Windows', function() end)
+    return
+  end
+end
```

Comment:
Not sure I can call pending in that context, https://ci.appveyor.com/project/equalsraf/neovim/build/1114/job/t2e8u4yf440fyuqp#L8667 - [helpers.lua](https://github.com/equalsraf/neovim/blob/5163c133a7b7e31e54e0eed75c4105e0f083fe71/test/functional/helpers.lua#L435)